api/management/api.go

```diff
@@ -98,19 +102,12 @@ func ConvertCredential(credKc *kc.CredentialRepresentation) CredentialRepresenta
 	var cred CredentialRepresentation
 	cred.Id = credKc.Id
 	cred.Type = credKc.Type
-	cred.Algorithm = credKc.Algorithm
+	cred.UserLabel = credKc.UserLabel
 	cred.CreatedDate = credKc.CreatedDate
-	if credKc.Config != nil {
-		var m map[string][]string
-		m = make(map[string][]string)
-		for _, key := range []string{"deviceInfo_Manufacturer", "deviceInfo_Model", "deviceInfo_Name", "deviceInfo_Plateform", "shortId", "usedChallenges", "availableChallenges", "cardStatus", "expirationDate"} {
-			value, ok := (*credKc.Config)[key]
-			if ok {
-				m[key] = value
-			}
-		}
-		cred.Config = &m
-	}
+	cred.CredentialData = credKc.CredentialData
+	cred.Temporary = credKc.Temporary
+	cred.Value = credKc.Value
```

Comment:
Do we want to transmit the Value ?

Reply:
Value is used to transmit raw password during password change